src/sota_tools/request_pool.cc

```diff
@@ -67,12 +69,12 @@ void RequestPool::LoopListen() {
   FD_ZERO(&fdread);
   FD_ZERO(&fdwrite);
   FD_ZERO(&fdexcept);
-  long timeoutms = 0;
+  int64_t timeoutms = 0;
   mc = curl_multi_timeout(multi_, &timeoutms);
```

Comment:
`curl_multi_timeout` also is defined to use `long`.

Reply:
ok